src/models/reserves.jl

```diff
@@ -1,6 +1,8 @@
 abstract type ReserveDirection end
 abstract type ReserveUp <: ReserveDirection end
 abstract type ReserveDown <: ReserveDirection end
+abstract type ReserveSupplementaryUp  <: ReserveDirection end
+abstract type ReserveSupplementaryDown  <: ReserveDirection end
```

Comment:
I see there's this comment below

> ```julia
> # Note that src/models/services_struct_types.jl and the testset
> # "Test struct type collections" in test/test_services.jl must be modified if any types are
> # added here.
> ```

As far as I can tell `src/models/services_struct_types.jl` no longer exists?
But the testset does: https://github.com/NREL-SIIP/PowerSystems.jl/blob/1b5beef78adbe7ec4285cddc3809696071c4177f/test/test_services.jl#L229-L242

Please can you update that test?

Also it would be good if we can update the comment to be correct about what changes to `src/` are required (and ideally move it above the code, as it's very easy to miss right now)

Reply:
This is a stale test and comment and I think both can be deleted.  The test is no longer doing anything useful.  I can take of this separately or if you want to do it here, that's OK with me.  Let me know.

Reply:
Thanks for the answer @daniel-thom.
As soon as we have more feedback on the [PSI PR](https://github.com/NREL-SIIP/PowerSimulations.jl/pull/484) about how is the best way to model our new reserve, I can make this change here.

Reply:
Thanks for making this cleaner.  We haven't always been great about fixing these sorts of things after revising implementations.